newrelic/core/agent_protocol.py

```diff
@@ -50,6 +50,131 @@
 _logger = logging.getLogger(__name__)
 
 
+# {return_value: ...} wrapper has been removed from each of these
+FAKE_AGENT_RESPONSE = {
+    'preconnect': {"redirect_host": "collector.newrelic.com"},
+    'connect': json_decode(r"""
+{
+    "agent_run_id": "xxx1",
```

Comment:
I removed everything that looked like an identifier or key and replaced it with `xxxN`.